lib/tasks/gettext.rake

```diff
@@ -17,9 +17,19 @@ namespace :gettext do
   end
 
   def files_to_translate
-    GettextSetup.config['source_files'].map do |p|
+    files = GettextSetup.config['source_files'].map do |p|
       Dir.glob(p)
     end.flatten
+    # check for optional list of files to exclude from string
+    # extraction
+    files_to_exclude = GettextSetup.config['exclude_files']
+    if (files_to_exclude)
```

Comment:
Yeah some people think it's confusing to do assignment inside an `if`, in any language, as it can be easy to confuse/typo "==" with "=". My personal stance is it's acceptable if you're _only_ using the variable within the enclosed code. I actually went back and forth on it here to begin with, because it's exactly the context I like to do that in.